src/WebJobs.Script.ConsoleHost/Verbs/KillVerb.cs

```diff
@@ -0,0 +1,72 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using NCli;
+
+namespace WebJobs.Script.ConsoleHost.Verbs
+{
+    [Verb(HelpText = "Kills running server instances of azf")]
```

Comment:
The string "azf" (or whatever the CLI ends up being called) should probably be moved to your constants.